custom/icds_reports/reports/disha.py

```diff
@@ -127,15 +112,18 @@ def build_dumps_for_month(month, rebuild=False):
             retry_count = 0
             while True:
                 try:
-                    dump.build_export_json()
+                    # force query to master after max retries
+                    query_master = retry_count == MAX_RETRY_COUNT
+                    dump.build_export_json(query_master=query_master)
                 except Exception as e:
```

Comment:
Can be make this exception handling more specific to expected failure?

Reply:
Since the exception is being checked on next line and is not swallowed up, it should be okay not to. Avoids the extra import.